src/components/home/home-page.tsx

```diff
@@ -16,18 +16,19 @@ const Root = styled.div`
   }
 `;
 
+const BgBeige = styled.div`
```

Comment:
isnt the section using the right background color depending on the nth selector ?

Reply:
yeah, ok reverted